Makefile

```diff
@@ -130,7 +130,7 @@ unit: fixtures ## Run unit tests (with coverage)
 .PHONY: integration
 integration: ## Run integration tests
 	$(call title,Running integration tests)
-	go test -tags=integration ./test/integration
```

Comment:
what happened here?

Reply:
essentially by renaming all files in the directory with `_test` we no longer need the `integration` build tag (🎉 ) which has been a common annoyance. I suggested this change on the last pairing session this morning.